tests/test_bot.py

```diff
@@ -0,0 +1,36 @@
+from __future__ import annotations
+
+import gymnasium as gym
+import numpy as np
+
+from minigrid.utils.bot import Bot
+
+
+def test_bot():
```

Comment:
What is the goal of the bot? Do we want the bot to solve every environment within the time limit?